package.json

```diff
@@ -19,16 +19,19 @@
     }
   ],
   "scripts": {
-    "start": "gulp",
+    "watch:server": "gulp watch-server",
```

Comment:
could we have a unified commands/scripts among our projects?

i have tried to make them unified but we are changing them time to time

cc/ @gokmen

Reply:
Agreed, it's better to keep with standards `npm start` should remain as is if needed updated to `gulp watch-server`

Reply:
The main reason for me to change this to `watch:server` is that this repository will create 2 different servers, 1 for api/backend (nodejs), and 1 for serving client (nginx). I was trying to be explicit.

But since we only need 1 npm script only for starting to server, i am changing this back to `npm  start`.